openshift-knative/eventing-kafka-broker
# Migrate current dispatcher to use `ReactiveKafkaConsumer` instead of Vertx KafkaConsumer. (#3177)

* Initial changes in the dispatcher

* Create Mock for Testing, and migrate some test

* OffseManagerTest migrated

* OrderedConsumer verticle test

* Set ConsumerRebalanceListener

* added Object not null check

* consumer verticle changed

* Fixed RecordDispatcherMutatorChainTest

* MockReac... poll() waits for timeout, fixes Unor..

* Move test Mock consumer to the right place

* Add dataplane support

* Remove Old Consumer Factory

* script file updated to use the dispatcher-vertx

* added missing licence header

* Benchmark

* Profiler Run changed to use vertx impl

* third party txt

* Use for instead of while
